DeepLearningSuite/DeepLearningSuiteLib/FrameworkEvaluator/KerasInferencer.cpp

```diff
@@ -172,7 +172,7 @@ int KerasInferencer::getKerasInferences(const cv::Mat& image, double confidence_
 		sizes[0] = image.rows;
 		sizes[1] = image.cols;
 	} else {
-		std::cout << "Invalid Image Passed" << '\n';
+		LOG(FATAL) << "Invalid Image Passed" << '\n';
```

Comment:
FATAL Errors kill the program, and that shouldn't happen, replace it with ERROR

Reply:
changed them , but
`make: /usr/local/Cellar/cmake/3.13.4/bin/cmake: No such file or directory`
is the reason for the failure .
How to fix this ?

Reply:
I will fix that. Travis may have recently updated cmake version, but the cached build shows that it was build using previous version of cmake which is not available now.
So, clearing cache will solve this issue.